src/Nancy.Swagger/Standalone.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Nancy.Swagger.Services;
+using Nancy.TinyIoc;
+using Swagger.ObjectModel;
+
+namespace Nancy.Swagger
+{
+    public static class Standalone
+    {
+        public static SwaggerRoot Generate(TinyIoCContainer container, NancyContext context)
```

Comment:
This class should not know about the TinyIoCContainer . You should just pass the ISwaggerMetadataProvider into this function and do the resolving before calling this function. 
I think it's possible to use other containers for the dependency injection, so we don't want to have a dependency on it.